Client/Frontend/Browser/Playlist/Managers & Cache/PlaylistCacheLoader.swift

```diff
@@ -397,7 +397,8 @@ class PlaylistWebLoader: UIView {
             $0.edges.equalToSuperview()
         }
         
-        if let browserController = (UIApplication.shared.delegate as? AppDelegate)?.browserViewController {
+        // TODO: REFACTOR to support multiple windows better
+        if let browserController = webView.currentScene?.browserViewController {
```

Comment:
Can you open an issue for this refactor and anywhere where we have this TODO around multi-window support ref that issue in its todo?